### PR TITLE
Feature/hook and winch tests

### PR DESCRIPTION
**Pull request by aschokking**

### CODE REVIEW

**Comment by aschokking** [comment]
@tfrancisco we also need a way to actually call those methods to update the distances.

**Comment by aschokking** [comment]
Okay these tests are all passing and this code should be relatively good to go.

File: CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package competition.subsystems.collector.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import competition.subsystems.collector.CollectorSubsystem;
+import competition.subsystems.hanger.HookSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class UpdateCollectorSensorsCommand extends BaseCommand {
```
Comment (WasabiFan) [inline CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java]: "update sensors" isn't particularly descriptive -- we should normalize on a phrase that we use for all of this type of command. I've been calling it "telemetry".

File: CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    
+    @Override
+    public void initialize() {
+        
+    }
+
+    @Override
+    public void execute() {
+        collectorSubsystem.isBallInCollector();
```
Comment (WasabiFan) [inline CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java]: Methods with side effects like this are scary -- the property(ies) should just be moved into the telemetry command.

File: CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java (hunk 1 above)
Comment (JohnGilb) [inline CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java]: +1 for good naming - it should be a separate PR, since we had already normalized on "UpdateSensors" across many subsystems.

File: CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java (hunk 2 above)
Comment (JohnGilb) [inline CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java]: Good idea
